Clinical-Genomics/scout
# Download2

This PR adds print functionality to the causatives view. Its done by using js dataTables. I added the js to the causatives.html but im not sure if thats where we want them. Maybe I should put them in some layout.html? This pr is made in responce to this [issue](https://github.com/Clinical-Genomics/scout/issues/1341)


**How to test**:
1. Install on clinical db by running:
`bash servers/resources/clinical-db.scilifelab.se/update-scout-stage.sh download2`
2. Go to : https://scout-stage.scilifelab.se 
3. Look at the causatives view. It looks a little different than before since everything is in a table now. See if ti is user friendly and nothing has got lost on the way. 


**Review:**
